pkg/errs-dictionary/errdict.go

```diff
@@ -0,0 +1,120 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package errdict
+
+import (
+	"strings"
+
+	"istio.io/pkg/structured"
+)
+
+const (
+	LikelyCauseFirstPrefix = "The likely cause is "
+	LikeyCauseSecondPrefix = "Another possible cause could be "
+)
+
+func fixFormat(s string) string {
+	s = strings.TrimSpace(s)
+	return strings.TrimSuffix(s, ".")
+}
+
+func formatCauses(causes ...string) string {
+	if len(causes) == 0 {
+		return ""
+	}
+	out := LikelyCauseFirstPrefix + fixFormat(causes[0]) + "."
+	for _, c := range causes[1:] {
+		out += LikeyCauseSecondPrefix + fixFormat(c) + "."
+	}
+	return out
+}
+
+// General messages
+const (
+	// Boilerplate messages applicable all over the code base.
+
+	// Action
+	ActionIfErrPersistsContactSupport          = "If this error persists, " + ActionContactSupport
+	ActionIfErrSureCorrectConfigContactSupport = "If you are sure your configuration is correct, " + ActionContactSupport
+	ActionContactSupport                       = "contact support if using managed Istio. Otherwise check if the issue is already fixed in " +
+		"another version of Istio at https://github.com/istio/istio/issues, raise an issue, or ask for help in " +
```

Comment:
nit: do we want to just like to some like like https://istio.io/latest/about/bugs/ so this isn't so verbose?

Reply:
good suggestion, thanks, done.